code/game/machinery/kitchen/processor.dm

```diff
@@ -216,6 +216,8 @@
 	return
 
 /obj/machinery/processor/attack_hand(var/mob/user as mob)
+	if(!ishuman(user) && !isrobot(user)) //No ghosts or mice putting people into the sleeper
```

Comment:
Oh right you meant that.

Because ghosts physically can't pick things up and insert them.  The only way to do that is via clickdragging and handled below it.

Because I don't know where attack_ghost() is defined.